app/assets/stylesheets/application.css

```diff
@@ -0,0 +1,454 @@
+/*
+ * This is a manifest file that'll be compiled into application.css, which will include all the files
+ * listed below.
+ *
+ * Any CSS (and SCSS, if configured) file within this directory, lib/assets/stylesheets, or any plugin's
+ * vendor/assets/stylesheets directory can be referenced here using a relative path.
+ *
+ * You're free to add application-wide styles to this file and they'll appear at the bottom of the
+ * compiled file so the styles you add here take precedence over styles defined in any other CSS
+ * files in this directory. Styles in this file should be added after the last require_* statement.
+ * It is generally better to create a new file per style scope.
+ *
+ *= require_tree .
+ *= require_self
+ */
+
+* {
+  font-family: 'Proxima Nova', sans-serif;
+}
+
+ul {
+  list-style: none;
+  padding-left: 0;
+}
+
+html {
+  background-color: rgb(230, 230, 230);
+}
+
+body {
+  margin: 0;
+}
+
+.home-icon i {
+  color: white;
```

Comment:
- [ ] Nice work including many functionalities (image storage, editing....), however, you need to follow the original design template as part of the requirements. check how your pages should look exactly

## The signup page 
<img width="321" alt="image" src="https://user-images.githubusercontent.com/65068771/219556247-24b9b07c-2d33-41a3-9fcc-a76f80ecaab7.png">

## the welcome page 
<img width="321" alt="image" src="https://user-images.githubusercontent.com/65068771/219556444-2876435d-f96a-4667-908e-22bdb385bd8c.png">

## Login page 
<img width="321" alt="image" src="https://user-images.githubusercontent.com/65068771/219556655-7b4b393b-c19b-473a-8f87-b3c098972e44.png">

## group page 
<img width="321" alt="image" src="https://user-images.githubusercontent.com/65068771/219556864-73616d4b-afca-4c79-874e-2ecbf166ed48.png">

## entities page 
<img width="321" alt="image" src="https://user-images.githubusercontent.com/65068771/219557461-b04def01-4025-4939-80e1-bf75d7237742.png">


Note: please make sure the design much the template by using the same  
* Colors
* Typography: font face, size, and weight.
* Layout: composition and space between elements.

>